Headers/DebugServer2/Base.h

```diff
@@ -45,7 +41,7 @@ typedef SSIZE_T ssize_t;
 #define OS_WIN32
 #elif defined(__FreeBSD__)
 #define OS_FREEBSD
-#elif defined(__APPLE__) && defined(TARGET_OS_MAC)
```

Comment:
Yeah but we'll have to do that when we actually target multiple apple targets. I'm almost certain that there's gonna be close to zero difference, given that the kernel is the same.